proposals/incubation/opencost.md

```diff
@@ -0,0 +1,189 @@
+
+As an existing Sandbox project this is already in place. All code is licensed Apache v2 and all documentation is Creative Commons Attribution 4.0.
+
+### **_Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope_**
+A list of public adopters can be found [here](https://github.com/opencost/opencost/blob/develop/ADOPTERS.MD). Additional adopters will be added shortly as this list was recently added.
+
+### **_Clear versioning scheme & release methodology_**
+
+As previously documented, OpenCost follows Semantic Versioning 2.0.0 and releases monthly.
+
+### **_Demonstrate a substantial ongoing flow of commits and merged contributions._**
+
+As previously documented, [OpenCost currently has 387 contributors and 67 committers](https://opencost.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1&var-period_name=Last%20decade&var-repogroup_name=OpenCost).
+
+Full details can be found at [CNCF devstats](https://opencost.devstats.cncf.io/) dashboards for OpenCost.
+
+### **_Security_**
+
+The security process for OpenCost is documented in the [SECURITY.md](https://github.com/opencost/opencost/blob/develop/SECURITY.md). Here is an abbreviated summary:
+* All code is version-controlled with known committers.
+* Dependencies are checked by Dependabot.
+* The current and previous releases are supported for security fixes.
+* Vulnerability reporting is handled through GitHub-provided private vulnerability reporting.
+* Vulnerabilities will be disclosed once confirmed and a public fix released.
+
+### **_Specifications must have at least one public reference implementation._**
+
+The Golang OpenCost project is the implementation of the [OpenCost specification](https://github.com/opencost/opencost/tree/develop/spec).
+
+## Statement on Alignment with the CNCF Mission
+
+OpenCost aligns with the mission of the CNCF by empowering organizations to monitor the cost of their cloud native environments and to make informed decisions on their optimizing their spend and application infrastructure.
+
+### Alignment with Other CNCF Projects
+
+OpenCost is currently aligned with the following CNCF projects:
+* Helm: OpenCost has an opencost-helm-chart repository that provides a Helm chart for deploying and upgrading OpenCost installations.
+* Kubernetes: OpenCost is installed on Kubernetes and relies on Kubernetes’ APIs for aligning Kubernetes resources with allocated cloud costs.
+* Prometheus: In default installations OpenCost stores metrics in Prometheus’ time series database and uses the Prometheus node-exporter to supplement monitoring data.
+* Cortex: Storage option used as an alternative to Prometheus, supported by the OpenCost Helm chart.
+* Thanos: Storage option used as an alternative to Prometheus, supported by the OpenCost Helm chart and used by Kubecost with their commercial implementation.
```

Comment:
We don't currently have any OTel integration plans, but we're open to working with the project. It's come up in conversations but we haven't seen a strong push yet. I'd be happy for suggestions where you think it would be most appropriate initially. I'll add it to the write-up.